zebra/rib.h

```diff
@@ -416,6 +416,7 @@ extern struct route_entry *rib_match_ipv6_multicast(vrf_id_t vrf_id,
 
 extern struct route_entry *rib_lookup_ipv4(struct prefix_ipv4 *p,
 					   vrf_id_t vrf_id);
+extern struct route_entry *rib_lookup(struct prefix *p, vrf_id_t vrf_id);
```

Comment:
Do we really need yet-another copy of the "lookup a route" logic for this purpose?

Reply:
There is only ipv4 lookup available, no ipv6 lookup was found. changing the prototype of "rib_lookup_ipv4" will lead to code impact as other functions use it. Instead of creating ipv6_lookup, the generic rib_lookup was added.

Reply:
looks like route_node_lookup() is widely used - won't that work?

Reply:
Finally we do not need another copy, as you are suggesting. The `rib_lookup_ipv4 `evolved to `rib_lookup` with generic ipv4 ipv6 functionality in new version.